storage/command_queue_test.go

```diff
@@ -171,3 +171,19 @@ func TestCommandQueueClear(t *testing.T) {
 		t.Fatal("commands should finish when clearing queue")
 	}
 }
+
+// TestCommandQueueExclusiveEnd verifies that an end key is treated as
+// an exclusive end when GetWait calculates overlapping commands. Test
+// it by calling GetWait with a command whose start key is equal to
+// the end key of a previous command.
+func TestCommandQueueExclusiveEnd(t *testing.T) {
```

Comment:
Sorry for the confusion. Yes, I should have added a test for the GC queue to make sure `gcArgs.Key` and `gcArgs.EndKey` are set correctly, but didn't do as it cannot be done easily without some refactoring to `process()` (I think).

`TestCommandQueueExclusiveEnd` just verifies that an end key of a command coming from `RequestHeader` is treated as an exclusive end. It doesn't exercise the change directly.